web/src/features/modals/InfoModal.tsx

```diff
@@ -35,38 +35,51 @@ export function InfoModalContent() {
       </div>
       <div>
         <Button
+          size="lg"
+          type="secondary-elevated"
+          backgroundClasses="my-2"
```

Comment:
Instead of adding the margin on each button here, you could add a `space-y-2` class on the `<div>` above it :)